src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccountDeployment.cs

```diff
@@ -61,6 +61,14 @@ public class NewAzureCognitiveServicesAccountDeploymentCommand : CognitiveServic
             Mandatory = true,
             ParameterSetName = DefaultParameterSet,
             ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Cognitive Services Deployment Sku.")]
+        public Sku Sku { get; set; }
```

Comment:
`Sku` is still shown as mandatory parameter. In addition, the parameter `DeploymentProperties` previously had position 3 and now updated to 4. This is also a breaking change. Can you please do the following?
- Move the `Sku` after `DeploymentProperties` and make it `Mandatory = false`
- Assign the `Position` of `Sku` with 4 if necessary

Reply:
I see, thank you, already pushed update with set Mandatory = false, and remove sku position.